README.md

```diff
@@ -640,6 +661,125 @@ or
 ecs-cli compose --ecs-params my-ecs-params.yml service up --launch-type FARGATE
 ```
 
+#### Using Route53 Service Discovery
+
+With the ECS CLI, you can create a Service that uses [Route53 auto naming for service discovery](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html). Service Discovery requires a Service Discovery Service, and a DNS Namespace. When you enable Service Discovery with the ECS CLI, a new Service Discovery Service is always created using CloudFormation. For the DNS Namespace, you have the option of using an existing public or private DNS Namespace, or letting the ECS CLI create a private DNS Namespace for you using CloudFormation. Creation of a Public DNS Namespaces is not supported with the ECS CLI. Please note that only a single DNS Namespace may be used with Service Discovery.
+
+##### Enabling Service Discovery
+
+###### Specifying Values
+
+There are many input parameters for Service Discovery, however, for most of them, the ECS CLI will choose reasonable defaults. Here are the default values and explanations listed with the ECS Params input schema:
```

Comment:
I would rephrase this to put a more positive spin -- e.g. "the ECS-CLI simplifies the use of Service Discovery by providing the following defaults, while still allowing maximum configurability..." or something like that.